appium/tests/specs/mock/setup/CheckRefreshKeyFromEKMFirst.spec.ts

```diff
@@ -12,35 +12,26 @@ import RefreshKeyScreen from "../../../screenobjects/refresh-key.screen";
 import BaseScreen from "../../../screenobjects/base.screen";
 import AppiumHelper from "../../../helpers/AppiumHelper";
 
-const ekmMockServer = 'http://127.0.0.1:8001/ekm';
-const successMessage = CommonData.refreshKeys.updatedSuccessfully;
-const processArgs = CommonData.processArgs;
 
 const goToKeysScreen = async () => {
   await MenuBarScreen.clickMenuIcon();
   await MenuBarScreen.clickSettingsButton();
   await SettingsScreen.clickOnSettingItem('Keys');
 }
 
-const logoutUserFromKeyScreen = async () => {
-  await KeysScreen.clickBackButton();
-  await MenuBarScreen.clickMenuIcon();
-  await MenuBarScreen.clickLogout();
-  await SplashScreen.checkLoginPage();
-}
-
 describe('SETUP: ', () => {
 
   it('app auto updates keys from EKM during startup with a pass phrase prompt', async () => {
```

Comment:
Let's rename the file too - `CheckRefreshKeyFromEkmWithPassPhrasePrompt.spec.ts`